sdk/metric/correct_test.go

```diff
@@ -39,19 +40,26 @@ var Must = metric.Must
 type correctnessBatcher struct {
 	t *testing.T
 
+	newAggCount int64
```

Comment:
This needs to be put first in the struct, otherwise it won't be aligned in 32bit builds. This is why the CI fails.